commands/penalty.js

```diff
@@ -0,0 +1,56 @@
+const Discord = require("discord.js");
+const mongoose = require("mongoose");
+const config = require(`../config.json`);
+const secrets = require(`../secrets.json`);
+const messageHandler = require(`../handlers/messageHandler.js`);
+const pointHandler = require(`../handlers/pointHandler.js`);
+const pointsChange = require("../models/addPoints.js");
+mongoose.connect(secrets.Mongo, {
+    useUnifiedTopology: true,
+    useNewUrlParser: true,
+});
+module.exports = {
+    key: "penalty",
+    name: "Penalty",
+    isModeratorCommand: true,
+    description: "This command penalizes the specified user by reducing their points by 1,000.",
+    syntax: `${config.moderatorPrefix}penalty <@user>`,
+    fieldDescriptions: [{ name: "`<@user>`", value: "This is the mention of the user the command should target." }],
+    registerWithHandlers() {
+        pointHandler.register(this);
+        messageHandler.register(this);
+    },
+    execute(prefix, msg, args) {
```

Comment:
```suggestion
execute(prefix, msg, args) {
```

Reply:
I set it up to support issue #67 where the signature will need to be the same for the execute command. If we change this spot to `msg` instead of `prefix, msg, args` the invocation in `main.js` will need to be updated too. I'll take it in whichever direction you desire.

Reply:
Oh, then leave it as is for now. We can always change it later if needed.